apiserver/src/error.rs

```diff
@@ -0,0 +1,26 @@
+use models::node::BottlerocketNodeError;
+
+use actix_web::error;
+use snafu::Snafu;
+
+/// The crate-wide result type.
+pub type Result<T> = std::result::Result<T, Error>;
+
+/// The crate-wide error type.
+#[derive(Debug, Snafu)]
+#[snafu(visibility = "pub")]
+pub enum Error {
+    #[snafu(display("Unable to create client: {}", source))]
```

Comment:
Nit: same as in other PRs, people usually ask me to put the error messages between `''`